public final class io.vertx.core.net.impl.NetSocketImpl$reportException__1322 extends java.lang.Object implements io.vertx.core.Handler
{
io.vertx.core.impl.ContextInternal cap0;
public static io.vertx.core.Handler bootstrap$(io.vertx.core.impl.ContextInternal)
{
io.vertx.core.net.impl.NetSocketImpl$reportException__1322 v1;
io.vertx.core.impl.ContextInternal v0;
v0 := @parameter0: io.vertx.core.impl.ContextInternal;
v1 = new io.vertx.core.net.impl.NetSocketImpl$reportException__1322;
specialinvoke v1.<io.vertx.core.net.impl.NetSocketImpl$reportException__1322: void <init>(io.vertx.core.impl.ContextInternal)>(v0);
return v1;
}
public void <init>(io.vertx.core.impl.ContextInternal)
{
io.vertx.core.net.impl.NetSocketImpl$reportException__1322 v0;
io.vertx.core.impl.ContextInternal v1;
v0 := @this: io.vertx.core.net.impl.NetSocketImpl$reportException__1322;
v1 := @parameter0: io.vertx.core.impl.ContextInternal;
specialinvoke v0.<java.lang.Object: void <init>()>();
v0.<io.vertx.core.net.impl.NetSocketImpl$reportException__1322: io.vertx.core.impl.ContextInternal cap0> = v1;
return;
}
public void handle(java.lang.Object)
{
io.vertx.core.net.impl.NetSocketImpl$reportException__1322 v0;
io.vertx.core.impl.ContextInternal v2;
java.lang.Object v1;
v0 := @this: io.vertx.core.net.impl.NetSocketImpl$reportException__1322;
v1 := @parameter0: java.lang.Object;
v2 = v0.<io.vertx.core.net.impl.NetSocketImpl$reportException__1322: io.vertx.core.impl.ContextInternal cap0>;
interfaceinvoke v2.<io.vertx.core.impl.ContextInternal: void reportException(java.lang.Throwable)>(v1);
return;
}
}